src/Illuminate/Translation/Translator.php

```diff
@@ -218,8 +218,8 @@ protected function makeReplacements($line, array $replace)
         $shouldReplace = [];
 
         foreach ($replace as $key => $value) {
-            $shouldReplace[':'.Str::ucfirst($key)] = Str::ucfirst($value);
-            $shouldReplace[':'.Str::upper($key)] = Str::upper($value);
+            $shouldReplace[':'.Str::ucfirst($key ?? '')] = Str::ucfirst($value ?? '');
```

Comment:
I don't think keys can ever be null?

Reply:
I only fixed the behavior that was previously in place. In any PHP version before 8.1, if `null` was used as a replacement it'd be casted to a string. Since PHP 8.1 throws a deprecation for this, we'll need to make sure to do this ourselves.

We can't change the behavior of handling `null` on 8.x or 9.x as that'd be a breaking change.